doc/changes.md

```diff
@@ -1,5 +1,71 @@
 # Changelog
 
+## Release v0.17.0 (2025/09/30)
+
+[![Digital Object Identifier for PyGMT v0.17.0](https://zenodo.org/badge/DOI/10.5281/zenodo.17156962.svg)](https://doi.org/10.5281/zenodo.17156962)
+
+### Highlights
+
+*
```

Comment:
I propose these three highlights. What do you think?
```suggestion
* 🎉 Seventeenth minor release of PyGMT 🎉
* Implement the new alias system towards to a Pythonic interface ([#3986](https://github.com/GenericMappingTools/pygmt/pull/3986), [#4000](https://github.com/GenericMappingTools/pygmt/pull/4000), [#3993](https://github.com/GenericMappingTools/pygmt/pull/3993))
* GMTDataArrayAccessor: Support applying grid operations on the current xarray.DataArray object ([#3854](https://github.com/GenericMappingTools/pygmt/pull/3854))
```

Reply:
Sounds good for me.